templates/includes/pagination.html

```diff
@@ -1,16 +1,17 @@
+{% load query_transform %}
 {% if is_paginated %}
   <ul class="pagination">
     {% if page_obj.has_previous %}
       <li class="page-item">
-        <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>
+        <a href="?{% query_transform request page=page_obj.previous_page_number %}" class="page-link">prev</a>
       </li>
     {% endif %}
     <li class="page-item active">
       <span class="page-link">{{ page_obj.number }} of {{ paginator.num_pages }}</span>
     </li>
     {% if page_obj.has_next %}
       <li class="page-item">
-        <a href="?page= {{ page_obj.next_page_number }}" class="page-link">next</a>
+        <a href="??{% query_transform request page=page_obj.next_page_number %}" class="page-link">next</a>
```

Comment:
There is a typo in the 'next' page link. It contains double question marks (`??`). It should be a single question mark (`?`) to correctly format the URL for the next page.